tools/build_defs/framework.bzl

```diff
@@ -73,6 +73,7 @@ CC_EXTERNAL_RULE_ATTRIBUTES = {
         ),
         mandatory = False,
         allow_files = True,
+        cfg = "host",
```

Comment:
you probably want `cfg = "exec"` 😄

Reply:
Do you know what version that was introduced? I was trying to find it but failed!

Reply:
Hmm, let me do some digging. https://docs.bazel.build/versions/master/skylark/rules.html#configurations is where I learned about this (just for documentations sake)

Reply:
And it appears to have first appeared in the [3.6.0](https://docs.bazel.build/versions/3.6.0/skylark/rules.html) documentation.